maths/harshad_numbers.py

```diff
@@ -35,95 +39,92 @@ def sum_of_digits(num: int, base_of_interest: int) -> str:
     >>> sum_of_digits(543, 1)
     Traceback (most recent call last):
         ...
-    ValueError: 'base_of_interest' must be between 36 and 2 inclusive
+    ValueError: 'base' must be between 36 and 2 inclusive
     >>> sum_of_digits(543, 37)
     Traceback (most recent call last):
         ...
-    ValueError: 'base_of_interest' must be between 36 and 2 inclusive
+    ValueError: 'base' must be between 36 and 2 inclusive
     """
 
-    if (base_of_interest > 36) or (base_of_interest < 2):
-        raise ValueError("'base_of_interest' must be between 36 and 2 inclusive")
+    if (base > 36) or (base < 2):
+        raise ValueError("'base' must be between 36 and 2 inclusive")

```

Comment:
```suggestion
    if base < 2 or base > 36:
        raise ValueError("'base' must be between 2 and 36 inclusive")
```
Parentheses are unnecessary, and it's kinda strange to read the range backward